bedrock/styleguide/templates/styleguide/base.html

```diff
@@ -54,7 +53,7 @@
 
 <h1>{% block content_title %}{{ page_title }}{% endblock %}</h1>
 
-{% block subnav %}
+{% block styleguide_subnav %}
```

Comment:
Any particular reason to change the name of this block? (the [styleguide home template](https://github.com/mozilla/bedrock/blob/master/bedrock/styleguide/templates/styleguide/home.html#L14) still references to the old name)